Jenkinsfile

```diff
@@ -177,6 +215,38 @@ stage('Build without sync') {
         stash includes: 'wrappers/build/**/*.dll', name: 'uwp-wrappers-nosync'
       }
     },
+    'macOS': {
+      nodeWithCleanup('osx') {
+        getArchive()
+
+        dir('wrappers') {
+          cmake 'build-osx', "${pwd()}/build", configuration
+        }
+
+        stash includes: "wrappers/build/Darwin/${configuration}/**/*", name: 'macos-wrappers-nosync'
```

Comment:
I thought about that, but the macOS and Linux branches complete before the Android branch does, and they're executed on the generic macOS and Docker nodes so removing them won't buy us anything in CI time.